class-debug-bar-cron.php

```diff
@@ -362,6 +362,7 @@ private function display_schedules() {
 
 
 			$schedules = wp_get_schedules();
+			uasort($schedules, array($this, 'schedules_sorting'));
```

Comment:
Please add spaces inside the braces.
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage